### PR TITLE
Doc update 002

### PR DESCRIPTION
**Pull request by ChrisBradley001**
This update contains some suggested language improvements.

### CODE REVIEW

**Comment by terracotta-org** [comment]
Can one of the admins verify this patch?

File: docs/src/docs/asciidoc/user/usermanaged.adoc [hunk 1]
```diff
@@ -25,36 +25,52 @@ Of course, if you find yourself requiring plenty of services, maybe the cache ma
 
 === API extensions
 
-While a `UserManagedCache` extends `Cache`, it offers additional methods:
+If you use a `UserManagedCache`, you need to configure all required services by hand.
+The `UserManagedCache` class extends the `Cache` class by offering additional methods:
+
+* init() - initializes the cache
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/usermanaged.adoc]: The methods listed here should be changed to code font by surrounding them with backticks.

Comment (ChrisBradley001) [inline docs/src/docs/asciidoc/user/usermanaged.adoc]: I have updated the PR to include backticks for the methods.

**Comment by ljacomet** [comment]
Looks good now - will reopen PRs for getting these changes on release/3.1 and master

**Pull request closed without merging**